### PR TITLE
Get Current Location action added

### PR DESCRIPTION
**Pull request by rowellx68**
_Please delete all sections in italics before submitting this PR._

**Checks**

- [x] I've checked there are no linting errors.
- [x] I've checked the code is still at 100% test coverage.

**Added Actions (if relevant)**

- getCurrentLocation

**Are you happy to be listed as a contributor on [Shortcuts.fun](https://shortcuts.fun)?**

_Yes_ 

**Any other information / comments**

_If this is your first time contributing, I'd love to hear what you thought about the process. Was it easy/hard to find the information you were looking for? Did the Contributing Guide make sense? What were your stumbling blocks? Basically, any and all feedback is encouraged!_

### CODE REVIEW

File: __tests__/actions/getCurrentLocation.spec.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { getCurrentLocation } from '../../src/actions';
+
+describe('getCurrentLocation function', () => {
+  it('is a function', () => {
+    expect(typeof getCurrentLocation).toBe('function');
+  });
+
+  it('builds a get current song action', () => {
```
Comment (kopischke) [inline __tests__/actions/getCurrentLocation.spec.ts]: s/song/location

Comment (rowellx68) [reply __tests__/actions/getCurrentLocation.spec.ts]: Doh 🤦‍♂️. Should be fixed now.

**Comment by joshfarrant** [comment]
Looks great, thanks @rowellx68!

Good spot too @kopischke - thanks!

I'm just going to take a look at why the build's failing (I think it's unrelated to this PR) then I'll merge it in shortly and add you on to Shortcuts.fun 🎉

**Comment by joshfarrant** [comment]
Sorted, thanks again!

**Pull request merged**